js/index.js

```diff
@@ -2,41 +2,57 @@
 
 function updateSubtotal(product) {
   console.log('Calculating subtotal, yey!');
-
-  //... your code goes here
+  const price = product.querySelector('.price span').innerHTML;
+  const quantity = product.querySelector('.quantity input').value;
+  const subtotal = price * quantity;
+  product.querySelector('.subtotal span').innerHTML = subtotal;
+  return subtotal
 }
 
-function calculateAll() {
-  // code in the following two lines is added just for testing purposes.
-  // it runs when only iteration 1 is completed. at later point, it can be removed.
-  const singleProduct = document.querySelector('.product');
-  updateSubtotal(singleProduct);
-  // end of test
-
+  
   // ITERATION 2
-  //... your code goes here
-
+function calculateAll() {
+  let subTotals = 0
+  const allProducts = document.querySelectorAll('.product')
+  allProducts.forEach(element => {
+    subTotals += updateSubtotal(element)
+  });
+  
   // ITERATION 3
-  //... your code goes here
+  const priceTotal = document.querySelector('#total-value span')
+  priceTotal.innerHTML = subTotals
```

Comment:
```suggestion
  priceTotal.innerText = subTotals
```
Lo mismo que antes :D